docs/src/pages/index.tsx

```diff
@@ -95,7 +91,7 @@ export default function Home({ location, path }: Props) {
                 iconRight={<ArrowRight />}
                 // @ts-expect-error asComponent has wrong type declaration
                 asComponent={GatsbyLinkToButton}
-                href="/getting-started/"
+                href="/getting-started/for-designers/"
```

Comment:
Many of these Tile changes appear to be reverting the changes we did for overview pages, not sure what happened on your end.